src/mp/util.cpp

```diff
@@ -37,14 +39,17 @@ size_t MaxFd()
 
 std::string ThreadName(const char* exe_name)
 {
-    char thread_name[17] = {0};
+    char thread_name[16] = {0};
```

Comment:
Note: This change looks right. Previous code was trying to make room for an extra null byte, but pthread_getname_np documentation does say 16 byte max includes the null byte like PR description says

Reply:
Right, the doc also says that the output name will be null terminated so it may seem ok to remove the initialization `= {0}` but I left it because I suspect that if `pthread_getname_np()` fails it may leave the name untouched.